src/modal/index.tsx

```diff
@@ -26,6 +26,7 @@ export interface ModalProps {
   readonly fullscreen?: boolean
   readonly displayDimmer?: boolean
   readonly closeButtonTitle?: string
+  readonly forwardedRef?: Ref<HTMLDivElement>
```

Comment:
Are you sure this is the right type? This seems pretty restrictive. Does `HTMLElement` work?

Reply:
It's not restrictive, it the right type as the ref is applied on a div.